src/libs/actions/Policy.js

```diff
@@ -403,6 +403,13 @@ function hideWorkspaceAlertMessage(policyID) {
     Onyx.merge(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, {alertMessage: ''});
 }
 
+/**
+ * @param {String} policyID
+ */
+function updateLastAccessedWorkspace(policyID) {
+    Onyx.set('lastAccessedWorkspace', policyID);
```

Comment:
Should be using `ONYXKEYS` constant here?

Reply:
Good call. Updated.